discovery/cds-services.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const router = express.Router();
+const serviceDefinitions = require('./service-definitions');
+const patientGreetingService = require('../services/patient-greeting');
+
+// Discovery Endpoint
+router.get('/', (request, response) => {
+  const discoveryEndpointServices = {
+    services: serviceDefinitions,
+  };
+  response.send(JSON.stringify(discoveryEndpointServices, null, 2));
```

Comment:
I think we should be using [response.json](http://expressjs.com/en/api.html#res.json) instead.

Reply:
Addressed in https://github.com/cds-hooks/sandbox-cds-services/pull/1/commits/5611a7fbba491d40267a41e095c0585c566fc7da